test/e2e/kubeproxy.go

```diff
@@ -38,7 +40,6 @@ import (
 const (
 	endpointHttpPort        = 8080
 	endpointUdpPort         = 8081
-	endpointHostPort        = 8082
```

Comment:
its a fair suggestion. I would keep the hostPort and do the NodeName in a separate PR.
Lets not combine things in the same PR.